cmd/util/const.go

```diff
@@ -0,0 +1,12 @@
+// This package is for command line specific utilities.
+package util
```

Comment:
Should we have a project global constant modules? I feel like version and client identifier are constants that could be used elsewhere.

Reply:
Agreed will add

Reply:
See [commit](https://github.com/pokt-network/pocket-core/commit/7c91b982f490fdb42aa4ab1f583dcb084621432a) for change